coordinator/src/schema.rs

```diff
@@ -239,6 +239,7 @@ diesel::table! {
         nostr -> Text,
         timestamp -> Timestamptz,
         fcm_token -> Text,
+        fcm_token_update -> Nullable<Timestamptz>,
```

Comment:
It looks like the `fcm_token_update` is always set, also if the `fcm_token` is empty. This would then rather represent the `last_login` or `updated_at` timestamp. I fine either way, but it would be nice to align the model correspondingly.